webapp/channels/src/components/rename_channel_modal/rename_channel_modal.tsx

```diff
@@ -291,13 +286,16 @@ export class RenameChannelModal extends React.PureComponent<Props, State> {
                                     defaultMessage='Display Name'
                                 />
                             </label>
-                            <LocalizedInput
+                            <input
                                 onChange={this.onDisplayNameChange}
                                 type='text'
                                 ref={this.getTextbox}
                                 id='display_name'
                                 className='form-control'
-                                placeholder={holders.displayNameHolder}
+                                placeholder={this.props.intl.formatMessage({
```

Comment:
so it will be like 
placeholder = formatMessage({id: 'rename_channel.displayNameHolder',
                                                  defaultMessage: 'Enter display name',})

Reply:
Done